controller/utils.go

```diff
@@ -61,6 +61,10 @@ func createOrUpdateAttachmentTicket(va *longhorn.VolumeAttachment, ticketID, nod
 }
 
 func handleReconcileErrorLogging(logger logrus.FieldLogger, err error, mesg string) {
+	if types.ErrorIsInvalidState(err) {
+		return
```

Comment:
Suggest still having a log but with a lower level like trace instead of info or debug. This could give us a chance to know why updating those settings, but they haven't taken effect in any error situation.

Reply:
As you wish.  As mentioned above, there is still logging, from the call to utilruntime.HandleError(err) in SettingController.handleErr, so it is not entirely silent.
We will be back to the 10 lines of logging instead of just two every hour, but at least they will be trace.

Reply:
yes, trace, so it will not be printed out by default. This is more for troubleshooting purposes.

Reply:
Sure.

Reply:
Thanks, @c3y1huang already updated this PR with this change.